app/Controller/ServersController.php

```diff
@@ -1042,35 +1042,35 @@ public function serverSettings($tab=false)
                 $this->set('branch', $gitStatus['branch']);
                 $this->set('commit', $gitStatus['commit']);
                 $this->set('latestCommit', $gitStatus['latestCommit']);
+
                 $phpSettings = array(
                     'max_execution_time' => array(
                         'explanation' => 'The maximum duration that a script can run (does not affect the background workers). A too low number will break long running scripts like comprehensive API exports',
                         'recommended' => 300,
-                        'unit' => false
+                        'unit' => 'seconds',
                     ),
                     'memory_limit' => array(
                         'explanation' => 'The maximum memory that PHP can consume. It is recommended to raise this number since certain exports can generate a fair bit of memory usage',
                         'recommended' => 2048,
```

Comment:
you could make a good case for changing the number to just 2 and using unit 'GB', right?